src/backends/beamcoder.ts

```diff
@@ -476,25 +483,23 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
         return packet as Packet;
     }
 
-    _resizeFrameData(frame): Uint8ClampedArray {
+    _setFrameDataToImageData(frame: beamcoder.Frame, target: Uint8ClampedArray) {
         const components = 4; // 4 components: r, g, b and a
```

Comment:
nit: but should components be passed in? I think it's characteristic to the frame and target and doesn't really seem like something this function should be defining?

Reply:
Maybe we can read that from the frame object somehow.

Reply:
I added RGBA_PIXEL_SIZE constant

Reply:
@stepancar so we could not find the frame component number? from `frame`?

Reply:
I didn't get we need to. This code works only with RGBA formate. in our logic it's hardcoded. If we want to support other formats - we need to extend public interface too.

But I don't think it makes sense. We use RGBA everywhere in our system

Reply:
True.